### ISSUE TITLE
adwita-qt crushes manuskript

### ISSUE THREAD

**Issue by josecupchik** [olivierkes/manuskript#883]
I have tried this option to get a dark option, but it crushes Manuskript when mouse was in action. I accept another option with gtk2, and it works better now.
Josef

**Comment by TheJackiMonster** [comment]
Yes, this is one huge problem we have with the Qt themes currently. The best thing I can do at the moment is suggesting to use the default themes (Fusion for example). The other themes may be broken or Qt doesn't handle them well. My idea to solve this is a transition from Qt to GTK (which makes sense for licensing as well) but I can't say how long that will take even though I have already started with it. ^^'
